content/develop/use/patterns/indexes/index.md

```diff
@@ -158,6 +178,14 @@ When representing much larger numbers, you need a different form of indexing
 that is able to index numbers at any precision, called a lexicographical
 index.
 
+## Time series indexes
+
+When you create a new time series using the [`TS.CREATE`]({{< baseurl >}}/commands/ts.create) command, you can associate one or more `LABELS` with it. Each label is a name-value pair, where the value can either be text or numeric. Labels serve as a secondary index that you can query using various time series commands.
```

Comment:
> 'Labels serve as a secondary index that you can query using various time series commands.'

Again, let's emphasize that this secondary index operates on multiple keys.

Consider changing to 
'Labels serve as a secondary index that allows you to execute queries on groups of time series.'